packages/common/src/models/Status.ts

```diff
@@ -4,3 +4,8 @@ export enum Status {
   SUCCESS = 'SUCCESS',
   ERROR = 'ERROR'
 }
+
+/** Detects if a status is in a non-terminal state */
```

Comment:
I like this, should circulate this in #eng-client. V minor nit with the name, is "unloaded" a conventional term? To me it sounds more like something was loaded and then removed. Maybe "statusIsNotLoaded" but not a strong opinion.

Reply:
I sat here for a good five minutes trying to name this because:
* I don't like things that have to be inverted to be useful (ex. `const isLoading = !isLoadedStatus()`)
* I was trying not to be to obscure with the terminology (`statusIsNonTerminal`)
Your suggestion is a good one!

Reply:
Oh wait, I also wanted it to indicate that `failed` is considered a loaded status. So I might go with NonTerminal in the end

Reply:
nit: `statusIsNotFinalized` or `statusIsPending`? pending sounds a bit like loading though...